### PR TITLE
New GDC example in the disco app card

### PR DESCRIPTION
**Pull request by creilly8**
## Description
Addresses issue #1848 . New example available here: http://localhost:3000/?appcard=disco. 
GDC list updated here: http://localhost:3000/?appcard=nciGdcBtn

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: client/src/app.js [hunk 1]
```diff
@@ -1352,6 +1352,24 @@ async function launchDisco(arg, app) {
 	if (!arg.genome) throw '"genome" parameter missing'
 	const genomeObj = app.genomes[arg.genome]
 	if (!genomeObj) throw 'unknown genome'
-	const _ = await import('#plots/disco/launch.adhoc.ts')
-	return await _.launch(arg.disco, genomeObj, app.holder0)
+	if (arg.disco.sample_id) {
```
Comment (xzhou82) [inline client/src/app.js:1355]: thanks. please document `.disco{}` parameter in https://github.com/stjude/proteinpaint/wiki/Embedding

File: front/public/cards/disco.json [hunk 2]
```diff
@@ -64,6 +65,27 @@
...
+            "label": "GDC Sample Data",
+            "urlparam": "?genome=hg38&dslabel=GDC&disco=1&sample=TCGA-AP-A059-01A",
+            "runargs": {
+                "noheader": true,
+                "nobox": true,
+                "genome": "hg38",
+                "disco": {
+                    "dslabel": "GDC",
+                    "genome": "hg38",
+                    "sample_id": "TCGA-AP-A059-01A"
```
Comment (xzhou82) [inline front/public/cards/disco.json:79]: the case is a hypermutator and slow to load. replace with TCGA-85-7710-01A

File: front/public/cards/nciGdcBtn.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+        "intro": "The NCI's Genomic Data Commmons <a href='https://gdc.cancer.gov/about-gdc' target='GDC homepage'>(GDC)</a> is a repository of harmonized datasets of large cancer cohorts. Following applications allow to access and visualize GDC datasets via on-the-fly query to the GDC API.<br><p style='margin: 10px 20px 0px;'><ul style='margin: 5px 15px;'><li><a href='/?appcard=gdcbam' target='GDC lollipop'>BAM slicing</a></li><li><a href='/?appcard=lollipop' target='GDC lollipop'>Coding mutation lollipop plot</a> (or use gene search box below)</li><li><a href='/?appcard=singlecell' target='GDC lollipop'>Single cell RNA-seq</a></li><li><a href='/?appcard=disco' target='GDC disco'>Disco plot</a></li><li><a href='/?appcard=scatterplot' target='GDC scatterplot'>Scatter plot</a></li></ul></p>",
```
Comment (xzhou82) [inline front/public/cards/nciGdcBtn.json:4]: replace samplescatter with oncomatrix. scatter plot using gdc gene exp example isn't added yet

File: front/public/cards/disco.json (hunk 2 above)
Comment (creilly8) [reply front/public/cards/disco.json]: @xzhou82 This is complete and the application list is updated as well.

**Review by xzhou82 — APPROVED** [review]

**Pull request merged**